release-content/0.14/release-notes/13057_Implement_volumetric_fog_and_volumetric_lighting_also_know.md

```diff
@@ -1 +1,13 @@
-TODO
+Not all fog is created equal.
+Bevy's existing implementation covers [distance fog](https://en.wikipedia.org/wiki/Distance_fog), which is fast, simple and not particularly realistic.
+
+In Bevy 0.14, this is supplemented with volumetric fog, based on [volumetric lighting](https://en.wikipedia.org/wiki/Volumetric_lighting), which simulates fog using actual 3D space, rather than simply distance from the camera.
+As you might expect, this is both prettier and more computationally expensive!
+
+In particular, this allows for the creation of stunningly beautiful "god rays" (light shafts) shining through the fog.
```

Comment:
Proper name is "crepuscular rays", fwiw.

Reply:
I think the term "god rays" should still be included here, because that's the more recognizable of the two terms. (Of course, why not both?)

Reply:
```suggestion
In particular, this allows for the creation of stunningly beautiful "god rays" (more properly, crepuscular rays) shining through the fog.
```